### PR TITLE
Add intial deploy script for the governor

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description
- Add deploy script for governor
- Add example constants for the governor

Related to #2

### CODE REVIEW

**Comment by alexkeating** [comment]
Current dependencies on/for this PR:
* main
  * **PR #6** <a href="https://app.graphite.dev/github/pr/ScopeLift/pooltogether-governor-upgrade/6" target="_blank">Graphite</a>  👈
    * **PR #7** <a href="https://app.graphite.dev/github/pr/ScopeLift/pooltogether-governor-upgrade/7" target="_blank">Graphite</a> 
      * **PR #9** <a href="https://app.graphite.dev/github/pr/ScopeLift/pooltogether-governor-upgrade/9" target="_blank">Graphite</a> 
        * **PR #11** <a href="https://app.graphite.dev/github/pr/ScopeLift/pooltogether-governor-upgrade/11" target="_blank">Graphite</a> 
          * **PR #12** <a href="https://app.graphite.dev/github/pr/ScopeLift/pooltogether-governor-upgrade/12" target="_blank">Graphite</a> 
            * **PR #13** <a href="https://app.graphite.dev/github/pr/ScopeLift/pooltogether-governor-upgrade/13" target="_blank">Graphite</a> 
              * **PR #14** <a href="https://app.graphite.dev/github/pr/ScopeLift/pooltogether-governor-upgrade/14" target="_blank">Graphite</a> 
                * **PR #15** <a href="https://app.graphite.dev/github/pr/ScopeLift/pooltogether-governor-upgrade/15" target="_blank">Graphite</a> 
                  * **PR #17** <a href="https://app.graphite.dev/github/pr/ScopeLift/pooltogether-governor-upgrade/17" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/pooltogether-governor-upgrade/6?utm_source=stack-comment).

File: script/Deploy.s.sol [hunk 1]
```diff
@@ -1,9 +1,23 @@
-// SPDX-License-Identifier: UNLICENSED
-
+// SPDX-License-Identifier: AGPL-3.0-only
```
Comment (alexkeating) [inline script/Deploy.s.sol:1]: *(arbitrary comment)*

- This is pretty much the same as gitcoin

File: script/DeployInput.sol [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+contract DeployInput {
+  // These are based on the current values for Governor Alpha
+  uint256 constant INITIAL_VOTING_DELAY = 1;
+  uint256 constant INITIAL_VOTING_PERIOD = 28_800;
+  uint256 constant INITIAL_PROPOSAL_THRESHOLD = 1000e18;
```
Comment (alexkeating) [inline script/DeployInput.sol:8]: I pulled these constants from their governor alpha

File: script/DeployInput.sol [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+contract DeployInput {
+  // These are based on the current values for Governor Alpha
+  uint256 constant INITIAL_VOTING_DELAY = 1;
```
Comment (apbendi) [inline script/DeployInput.sol]: Whatever number we ultimately use for this, we definitely do *not* want it to be `1`, as that opens Governance up to multi-block MEV attacks. Why don't we use 3,600 for now (12 hours) and add a comment documenting that we need to get the updated values for all of these from the PT folks.

File: script/Deploy.s.sol (hunk 1 above)
Comment (apbendi) [reply script/Deploy.s.sol:1]: Let's include updates to the README explaining how to use the script in the PR, similar to GC

File: script/Deploy.s.sol [hunk 4]
```diff
@@ -1,9 +1,23 @@
...
-import "forge-std/Script.sol";
+import {Script} from "forge-std/Script.sol";
+import {DeployInput} from "script/DeployInput.sol";
+import {PooltogetherGovernor} from "src/PooltogetherGovernor.sol";
+
+contract Deploy is DeployInput, Script {
+  uint256 deployerPrivateKey;
+
+  function setUp() public {
+    deployerPrivateKey = vm.envUint("DEPLOYER_PRIVATE_KEY");
```
Comment (apbendi) [inline script/Deploy.s.sol:12]: we should have a .env.template file with this var represented there

**Review by apbendi — CHANGES_REQUESTED** [review]
Thanks @alexkeating, all small stuff!

File: script/Deploy.s.sol (hunk 1 above)
Comment (alexkeating) [reply script/Deploy.s.sol:1]: https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/6/commits/7f44d8e4219d164eb2ebb130aadd53a3b50a4c81

File: script/Deploy.s.sol (hunk 4 above)
Comment (alexkeating) [reply script/Deploy.s.sol:12]: https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/6/commits/7f44d8e4219d164eb2ebb130aadd53a3b50a4c81

File: script/DeployInput.sol (hunk 3 above)
Comment (alexkeating) [reply script/DeployInput.sol]: https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/6/commits/17714a37ded1153a5ad8e9fd59a2a3cf50f3c817 Added a comment and updated

**Review by apbendi — APPROVED** [review]
Nice work @alexkeating!

**Pull request merged**